test/fixture/vcr_cassettes/incidents_v1#list.json

```diff
@@ -0,0 +1,40 @@
+[
+  {
+    "request": {
+      "body": "",
+      "headers": {
+        "Accept": "application/json",
+        "Content-Type": "application/json",
+        "User-agent": "IncidentIo/elixir",
+        "Authorization": "token yourtokengoeshere"
+      },
+      "method": "get",
+      "options": [],
+      "request_body": "",
+      "url": "https://api.incident.io/v1/incidents?page_size=&after=&status="
```

Comment:
🤔 These empty query parameters are odd. They should have been dropped if with nil values or if none are provided.